### ISSUE TITLE
histogramdd  with sparse= True  fails with input tensor = vector (dim 1)

### ISSUE THREAD

**Issue by pierrebaudot** [francois-rozet/torchist#6]
### Description
First of all THANK YOU very much for this nice work: it helps me a lot, and I though contacting you because given your code you likely work on closely related topic with information topology problematic i work on. The problem (if it is not a misunderstanding of mine) encountered is:
if a 1 dimensional tensor (vector) is given in input to Histogramdd  with sparse= True  then one get the error:
"line 194, in histogramdd
    hist = torch.sparse_coo_tensor(idx.t(), values, shape)
RuntimeError: number of dimensions must be sparse_dim (10) + dense_dim (0), but got 1"

### Reproduce

A minimal working example demonstrating the current behavior.

```python
import torch
import torchist

x = torch.rand(100, 1).cuda()

hist = torchist.histogramdd(x, bins=10, low=0.0, upp=1.0, sparse = True)

```

### Expected behavior

One could expect a sparse vector in output

**Comment by francois-rozet** [comment]
Hello @pierrebaudot, and thank you for the bug report. This is indeed a bug, and I can reproduce it with your code snippet. I have fixed the issue in [release v0.2.3](https://github.com/francois-rozet/torchist/releases/tag/v0.2.3). The issue came from the special handling of the $D = 1$ case, which was not necessary anyway.

If the new release works for you, you can close this issue.

**Comment by pierrebaudot** [comment]
Thank you with a bit of delay.... It works good for me. Hope to see you one day.
